OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -253,13 +286,13 @@ private async void InitializeLibVLC(InitializedEventArgs eventArgs)
             // Create LibVLC instance.
             LibVLC = new LibVLC(eventArgs.SwapChainOptions);
             MediaPlayer = new MediaPlayer(LibVLC);
```

Comment:
I want to test if this will work once I get back home.
```suggestion
            _mediaPlayer = new MediaPlayer(LibVLC);
```